examples/cli_workflow/nmf_heart_config.yaml

```diff
@@ -0,0 +1,100 @@
+# lightning.pytorch==2.2.1
```

Comment:
Let's include only one config file, and let's have it be the config file that is used by the notebook tutorial for cNMF.